util/profile/trackerRecorder.go

```diff
@@ -31,24 +31,38 @@ func HeapProfileForGlobalMemTracker(d time.Duration) {
 	for {
 		select {
 		case <-t.C:
-			err := heapProfileForGlobalMemTracker()
+			err := heapProfileForPreparePlanCache()
 			if err != nil {
 				log.Warn("profile memory into tracker failed", zap.Error(err))
 			}
 		}
 	}
 }
 
-func heapProfileForGlobalMemTracker() error {
-	bytes, err := col.getFuncMemUsage(kvcache.ProfileName)
+// Profile name for prepare plan cache.
+const (
+	PreparePlanCacheKey   = "github.com/pingcap/tidb/planner/core.NewPSTMTPlanCacheKey"
+	PreparePlanCacheValue = "github.com/pingcap/tidb/planner/core.NewPSTMTPlanCacheValue"
+)
+
+func heapProfileForPreparePlanCache() error {
+	total := int64(0)
+	bytes, err := col.getFuncMemUsage(PreparePlanCacheKey)
+	if err != nil {
+		return err
+	}
+	total += bytes
+	bytes, err = col.getFuncMemUsage(PreparePlanCacheValue)
```

Comment:
`getFuncMemUsage` will  parse the memory heap and travel the heap structure tree each time. I think we can leave a `TODO` here to improve `getFuncMemUsage` by receiving multiple function keys.

Reply:
Addressed.